git_machete/cmd.py

```diff
@@ -3141,19 +3141,22 @@ def print_line_prefix(b_: str, suffix: str) -> None:
           <b>-f, --fork-point=<fork-point-commit></b>    Specifies the alternative fork point commit after which the rebased part of history is meant to start.
     """,
     "show": """
-        <b>Usage: git machete show <direction></b>
+        <b>Usage: git machete show [--branch=<branch>] <direction></b>
         where <direction> is one of: `c[urrent]`, `d[own]`, `f[irst]`, `l[ast]`, `n[ext]`, `p[rev]`, `r[oot]`, `u[p]`
 
         Outputs name of the branch (or possibly multiple branches, in case of `down`) that is:
 
         * `current`: the current branch; exits with a non-zero status if none (detached HEAD)
```

Comment:
Nit: what happens for if `--branch=something current`?

Reply:
Ok, AFAICS `current` takes precedence / `-b` is ignored... I think such combination can result in an error